internal/client-go/api_courier.go

```diff
@@ -131,7 +131,7 @@ func (a *CourierApiService) GetCourierMessageExecute(r CourierApiApiGetCourierMe
 	if r.ctx != nil {
 		// API Key Authentication
 		if auth, ok := r.ctx.Value(ContextAPIKeys).(map[string]APIKey); ok {
-			if apiKey, ok := auth["oryAccessToken"]; ok {
+			if apiKey, ok := auth["oryApiKey"]; ok {
```

Comment:
Hm this does cause breaking changes :thinking: No idea tbh on how we can patch this to fall back to the old key if it is not provided...
I guess there is no way, breaking changes in the SDK are unavoidable with these sh... generator templates in general, but old SDK versions should still be compatible with the server.

Reply:
The question is whether we have to break this, just to update the naming of this key? Maybe we can fix this in all services in one go etc

Reply:
This also means that we have to update all the docs for the Go examples btw